examples/dom/color_info_sorted_2d.ipp

```diff
@@ -33,22 +32,22 @@ std::vector<std::vector<ftxui::ColorInfo>> ColorInfoSorted2D() {
               [](const ftxui::ColorInfo& A, const ftxui::ColorInfo& B) {
                 return A.value < B.value;
               });
-    for (size_t i = 1; i < column.size(); ++i) {
-      size_t best_index = i;
+    for (int i = 0; i < int(column.size()) - 1; ++i) {
+      int best_index = i + 1;
       int best_distance = 255 * 255 * 3;
-      for (size_t j = i; j < column.size(); ++j) {
-        int dx = column[i - 1].red - column[j].red;
-        int dy = column[i - 1].green - column[j].green;
-        int dz = column[i - 1].blue - column[j].blue;
+      for (int j = i + 1; j < column.size(); ++j) {
+        int dx = column[i].red - column[j].red;
+        int dy = column[i].green - column[j].green;
+        int dz = column[i].blue - column[j].blue;
         int distance = dx * dx + dy * dy + dz * dz;
         if (best_distance > distance) {
           best_distance = distance;
           best_index = j;
         }
       }
-      std::swap(column[i], column[best_index]);
+      std::swap(column[i + 1], column[best_index]);
     }
   }
 
-  return info_columns;
+  return std::move(info_columns);
```

Comment:
writing std move here disables copy elision and forces a call to the move constructor instead (as far as i know)

Reply:
You are right ;-)

Reply:
then why did you revert it?